src/web/client/WebExtensionContext.ts

```diff
@@ -391,6 +391,7 @@ class WebExtensionContext implements IWebExtensionContext {
                     '',
                     response?.status.toString()
                 );
+                return;
```

Comment:
infact same for these return - just throw the response as error and catch will do the rest! We can remove this extra API logging from these 3 functions